### PR TITLE
Add default arguments to function explorer, some fixes

### LINKED ISSUE #123: 🐛 [BUG] - streamlit badges example is broken
### Description

nothing big, but the bottom of the streamlit badges is broken! was taking a look tonight and noticed

### Reproduction steps

```bash
head over to https://extras.streamlit.app/Badges and you'll notice
```

### Screenshots

_No response_

### Logs

_No response_

### Version of streamlit

na

### Version of streamlit-extras

na

### LINKED ISSUE #124: ✨ [IDEA] - Function explorer can take default arg values
### Description

In https://github.com/arnaudmiribel/streamlit-extras/blob/main/src/streamlit_extras/function_explorer/__init__.py#L30, it'd be nice to be able to pass default values for args, so that widgets actually work in the default state when using function_explorer

### Minimal code for the extra

_No response_

### Screenshots

_No response_

### LINKED ISSUE #126: 🐛 [BUG] - Error displayed in the `App logo` Extra.
### Description

Description
Error displayed in the `App logo` Extra.

image

Reproduction steps
See below

Logs
```
FileNotFoundError: This app has encountered an error. The original error message is redacted to prevent data leaks. Full error details have been recorded in the logs (if you're on Streamlit Cloud, click on 'Manage app' in the lower right of your app).
Traceback:
File "/app/streamlit-extras/gallery/streamlit_patches.py", line 283, in _run_script
    c()
File "/app/streamlit-extras/gallery/streamlit_app.py", line 223, in page_content
    function_explorer(func=func)
File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/runtime/metrics_util.py", line 311, in wrapped_func
    result = non_optional_func(*args, **kwargs)
File "/home/appuser/venv/lib/python3.9/site-packages/streamlit_extras/function_explorer/__init__.py", line 115, in function_explorer
    func(**inputs)
File "/home/appuser/venv/lib/python3.9/site-packages/streamlit_extras/app_logo/__init__.py", line 24, in add_logo
    logo = f"url(data:image/png;base64,{base64.b64encode(Path(logo_url).read_bytes()).decode()})"
File "/usr/local/lib/python3.9/pathlib.py", line 1259, in read_bytes
    with self.open(mode='rb') as f:
File "/usr/local/lib/python3.9/pathlib.py", line 1252, in open
    return io.open(self, mode, buffering, encoding, errors, newline,
File "/usr/local/lib/python3.9/pathlib.py", line 1120, in _opener
    return self._accessor.open(self, flags, mode)
```

Version of streamlit
na

Version of streamlit-extras
na

### Reproduction steps

```bash
1. Go to 'https://extras.streamlit.app/App%20logo'
2. See error
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Logs

_No response_

### Version of streamlit

na

### Version of streamlit-extras

na

### PR DESCRIPTION
**Pull request by blackary**
Fixes #126
Fixes #124 
Fixes #123 

Also fixed the images of flags

### CODE REVIEW

File: src/streamlit_extras/app_logo/__init__.py [hunk 1]
```diff
@@ -29,9 +29,12 @@ def add_logo(logo_url: str, height: int = 120):
             [data-testid="stSidebarNav"] {{
                 background-image: {logo};
                 background-repeat: no-repeat;
-                padding-top: {height - 40}px;
+                padding-top: {height + 30}px;
```
Comment (blackary) [inline src/streamlit_extras/app_logo/__init__.py:32]: I think the restyling of the sidebar must have changed how high it goes, or maybe I never tested scrolling with a long list, but this prevents the list of pages from overlapping the logo

**Review by arnaudmiribel — COMMENTED** [review]
Thanks @blackary - I deployed a staging app but it does not look like your updates to the function explorer get reflected in the app 🤔 See https://arnaudmiribel-streamlit-extras-gallerystreamlit-app-add--63bt9i.streamlit.app/Badges

**Comment by blackary** [comment]
> Thanks @blackary - I deployed a staging app but it does not look like your updates to the function explorer get reflected in the app 🤔 See https://arnaudmiribel-streamlit-extras-gallerystreamlit-app-add--63bt9i.streamlit.app/Badges

Can you make that app public?

**Comment by arnaudmiribel** [comment]
> > Thanks @blackary - I deployed a staging app but it does not look like your updates to the function explorer get reflected in the app 🤔 See https://arnaudmiribel-streamlit-extras-gallerystreamlit-app-add--63bt9i.streamlit.app/Badges
> 
> 
> 
> Can you make that app public?

It is!

**Comment by blackary** [comment]
@arnaudmiribel Can you change the app from this branch to point to `gallery_app.py`? I figured that if I moved the app out of the `gallery/` folder, it could actually install the local version of streamlit extras, so the gallery app would always be in-sync with the library.

**Comment by arnaudmiribel** [comment]
> @arnaudmiribel Can you change the app from this branch to point to `gallery_app.py`? I figured that if I moved the app out of the `gallery/` folder, it could actually install the local version of streamlit extras, so the gallery app would always be in-sync with the library.

Sure, here's the new link (I can't change the target of an app that's already deployed): https://arnaudmiribel-streamlit-extras-gallery-app-add-default-a-7xztw6.streamlit.app/

**Comment by arnaudmiribel** [comment]
@blackary Want me to review this? Let me know how to best do it

**Comment by sfc-gh-amiribel** [comment]
Looks like "App logo" works but not "Badges" or "Stateful button"???

**Comment by arnaudmiribel** [comment]
@blackary closing for now after inactivity

**Pull request closed without merging**